datadog/threadstats/base.py

```diff
@@ -298,6 +313,14 @@ def flush(self, timestamp=None):
             else:
                 log.debug("No metrics to flush. Continuing.")
 
+            count_dists = len(dists)
+            if count_dists:
+                self.flush_count += 1
```

Comment:
Maybe track metric and distribution flushes separately instead of combining the counts?

Reply:
it includes events so going to keep it